symplyphysics/laws/thermodynamics/euler_relations/internal_energy_formula.py

```diff
@@ -0,0 +1,72 @@
+from sympy import Eq
```

Comment:
How could user find that the law of interest is in Euler relations? Avoid marketing names.

Reply:
I can't really see any characteristic property of these relations, save they are derived using the Euler's homogeneous function theorem.